README.md

```diff
@@ -1,21 +1,7 @@
 ![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
 
-# MODULE_NAME Service
+# `svc-pricing`
 
-*TODO after cloning:*
-
-1. *Replace the repository name for each:*
-
-![Rust
-Checks](https://github.com/arrow-air/svc-template-rust/actions/workflows/rust_ci.yml/badge.svg?branch=main)
```

Comment:
Would you like to keep the badges?

Reply:
[6e2fef7](https://github.com/Arrow-air/svc-pricing/pull/1/commits/6e2fef790be7f2c47a0cf076a1de325654b639e2)